src/OpenDSSDirect.jl

```diff
@@ -1,30 +1,118 @@
-__precompile__()
 
 module OpenDSSDirect
 
-include("core.jl")
-include("api.jl")
-include("repl.jl")
+export LIBRARY
```

Comment:
Don't export this. It's an internal thing.

Reply:
Fixed in 3259111 and f1bc5ec.